README.md

```diff
@@ -1,2 +1,105 @@
 # terraform-provider
 Ably Terraform Provider
+
+# [Ably](https://www.ably.com)
```

Comment:
Looking at the rendered view, this heading feels superfluous and interrupts reader flow. I would remove it.

Reply:
Fixed in 38fb4f7c9637b193c15452027b0e9b184d62762a